tools/razzle.cmd

```diff
@@ -84,6 +18,20 @@ set OPENCON=%OPENCON_TOOLS:~0,-7%
 rem Add nuget to PATH
 set PATH=%PATH%%OPENCON%\dep\nuget;
 
+rem Run nuget restore so you can use vswhere
+nuget restore %OPENCON%
+
+rem Find vswhere
+rem from https://github.com/microsoft/vs-setup-samples/blob/master/tools/vswhere.cmd
+for /f "usebackq delims=" %%I in (`dir /b /aD /o-N /s "%~dp0..\packages\vswhere*"`) do (
+    for /f "usebackq delims=" %%J in (`where /r "%%I" vswhere.exe 2^>nul`) do set VSWHERE=%%J
+)
+
+rem Add path to MSBuild Binaries
+for /f "usebackq tokens=*" %%i in (`%VSWHERE% -latest -requires Microsoft.Component.MSBuild -find MSBuild\**\Bin\MSBuild.exe`) do set MSBUILD=%%i

```

Comment:
You should add `-products *` if you want to also support Visual Studio Build Tools. By default, vswhere only supports finding Community, Professional, and Enterprise - our traditional VS SKUs.